src/test/resources/json/graphql_valid_ffid_multiple_records_expected_2.json

```diff
@@ -0,0 +1,20 @@
+{
+  "query": "mutation addFFIDMetadata($addFFIDMetadataInput:FFIDMetadataInput!){addFFIDMetadata(addFFIDMetadataInput:$addFFIDMetadataInput){fileId}}",
+  "variables": {
+    "addFFIDMetadataInput": {
+      "fileId": "44aaeffb-41b6-4250-97d5-f8c4aebf00a1",
```

Comment:
Would it be possible for expected_1 and expected_2 to contain different files, or some kind of different data? At the moment, it could be sending the same result twice and the tests wouldn't catch it.

Reply:
Yeah, good point. I've added in another input file for each update type and changed the second multiple expected file to match the new file.